package/MDAnalysis/core/topologyattrs.py

```diff
@@ -1799,6 +1800,137 @@ def n_fragments(self):
         ('n_fragments', property(n_fragments, None, None,
                                  n_fragments.__doc__)))
 
+    def unwrap(self, box=None, compound='fragments', reference='com',
+               inplace=True):
+        """Move atoms in this group so that bonds within the
+        group's compounds aren't split accross periodic boundaries.
+
+        Parameters
+        ----------
+        box : array_like or None, optional
+            The unitcell dimensions of the system, which can be orthogonal or
+            triclinic and must be provided in the same format as returned by
+            :attr:`MDAnalysis.coordinates.base.Timestep.dimensions`:\n
+            ``[lx, ly, lz, alpha, beta, gamma]``.
+            If ``None``, uses the dimensions of the current timestep.
+        compound : {'group', 'segments', 'residues', 'molecules', 'fragments'}, optional
+            Which type of components to unwrap. Note that, in any case, all
+            atoms within each compound must be interconnected by bonds, i.e.,
+            compounds must correspond to (parts of) molecules.
+        reference : {'com', 'cog', None}, optional
+            If ``'com'`` (center of mass) or ``'cog'`` (center of geometry), the
+            unwrapped compounds will be shifted so that their individual
+            reference point lies within the primary unit cell.
+            If ``None``, no such shift is performed.
+        inplace : bool, optional
+            If ``True``, coordinates are modified in place.
+
+        Returns
+        -------
+        coords : numpy.ndarray
+            Unwrapped atom coordinate array of shape ``(n, 3)``.
+
+        Note
+        ----
+        * This is the inverse transformation of packing atoms into the primary
```

Comment:
There's some ascii in make_whole which you can steal for this. Maybe also a seealso for pack_into_box (which maybe we can rename wrap?)

Reply:
`*Group.wrap()` already exists and extends the functionality of `*Group.pack_into_box()`. I'm in favor of deprecating `pack_into_box()`.

Reply:
I just realized that `pack_into_box()` can do out-of-place packing, while `wrap()` always acts inplace. That should be changed. Furthermore, `wrap()` potentially acts on atoms outside of the calling group. That's dangerous and should be changed as well.